MAUI/Picker/dealing-with-column-header.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: post
+title: Column header with .NET MAUI Picker control | Syncfusion
+description: Learn here all about column header with Syncfusion .NET MAUI Picker (SfPicker) control.
+platform: maui
+control: SfPicker
+documentation: ug
+---
+
+# Dealing with Column Header in .NET MAUI Picker (SfPicker)
+
+This section explains about the column header customization of picker control.
+
+## Column header view
+
+SfPicker allows enabling or disabling the column header section by setting  `SfPicker.ColumnHeaderView.Height` property grater than 0. Default value of `SfPicker.ColumnHeaderView.Height` property is 0.
```

Comment:
SfPicker allows enabling or disabling the column header section by setting the `SfPicker.ColumnHeaderView.Height` property greater than 0. The default value of the `SfPicker.ColumnHeaderView.Height` property is 0.

Reply:
Content modified